src/markdown-pages/docs/cli-command-reference/runs-submit-local.md

```diff
@@ -11,7 +11,7 @@ Local runs do not benefit from the features that are provided when running tests
 
 ## Working with the `runs submit local` command
 
-To use the `runs submit local` command, the `$JAVA_HOME` variable must be set to reference the JVM in which you want the test to run. This is because the local java run-time environment is used to launch the test locally. The level of Java must match the supported level of the Galasa version that is being launched. Use the `galasactl --version` command to find the galasactl tool version.
+To use the `runs submit local` command, the `$JAVA_HOME` variable must be set to reference the JVM in which you want the test to run. This is because the local java run-time environment is used to launch the test locally. The level of Java must match the supported level of the Galasa version that is being launched. Use the `galasactl --version` or `galasactl.exe --version` command to find the galasactl tool version.
```

Comment:
To check that JAVA_HOME is set correctly, the tool checks that `$JAVA_HOME/bin/java` exists in unix/mac, and `%JAVA_HOME%\bin\java.exe` exists on Windows.

Reply:
added